src/uct/base/uct_iface.c

```diff
@@ -911,3 +927,19 @@ uct_base_ep_connect_to_ep(uct_ep_h tl_ep,
 
     return uct_ep_connect_to_ep_v2(tl_ep, device_addr, ep_addr, &param);
 }
+
+ucs_status_t
+uct_iface_is_reachable_v2_wrapper(const uct_iface_h tl_iface,
```

Comment:
I guess we would remove this wrapper when all transports support is_reacahable_v2?
why need to pass function pointer iface_is_reachable_v2? can't we just call uct_iface_is_reachable_v2() on the tl_iface ?

Reply:
Evgney also posted similar comments. And I tried. Looks it has problem. Some test case will be failed. 

```
/scrap/azure/agent-06/AZP_WORKSPACE/2/s/contrib/../src/uct/ib/dc/dc_mlx5.c: [ uct_dc_mlx5_iface_is_reachable_v2() ]
      ...
      898     uct_dc_mlx5_iface_t UCS_V_UNUSED *iface;
      899 
      900     iface = ucs_derived_of(tl_iface, uct_dc_mlx5_iface_t);
==>   901     ucs_assert_always(iface_addr != NULL);
```